changelog.md

```diff
@@ -1,5 +1,61 @@
 # Changelog
 
+
+## 0.15.0
+
+This release is brings download progress bars for file downloads
+and does an overhaul on printing.
+
+* TODO add commits merged after 4ca54cc
+
+* Initial support for IPP / driverless printing #2332
+* Use Breaks to remove old print-related dependencies #2434
+* Replace XPP with GTK print dialog #2411
+* Display download progress for file downloads
+  * Display download progress for file downloads #2327
+  * Lazy-load DownloadConversation enabled state #2428
+  * Make progress bars fit in narrow windows #2427
+  * Visually update download progress bars every second #2426
+  * Fix progress bars when entire conversation is downloaded #2424
+  *(TODO: is sub issue?) fix(`_streaming_download`): handle retry-level JSON error responses #2329
+
+* Dependencies:
+  * update to jinja2 3.1.6 for CVE-2025-27516 #2410
+  * Bump openssl from 0.10.66 to 0.10.70 #2374
+  * Upgrade openssl (RUSTSEC-2025-0022) and tokio (RUSTSEC-2025-0023) #2414
+  * Upgrade Rust toolchain to 1.84.1 #2379
+  * TODO: Dev / build deps?
```

Comment:
Yeah, I think these can be collapsed a lot. Note that the `url` and `six` ones are prod dependencies, not dev